src/index.test.js

```diff
@@ -1,7 +1,20 @@
-import { describe, it, expect } from 'vitest';
+import { describe, it, expect } from "vitest";
+import { dollarFormat, kebabCase } from "$lib/utils/index.js";
 
-describe('sum test', () => {
-	it('adds 1 + 2 to equal 3', () => {
-		expect(1 + 2).toBe(3);
-	});
+describe("sum test", () => {
+  it("adds 1 + 2 to equal 3", () => {
+    expect(1 + 2).toBe(3);
+  });
+});
+
+describe("kebab-case test", () => {
+  it("converts a string to kebab-case", () => {
+    expect(kebabCase("Hello World")).toBe("hello-world");
+  });
+});
+
+describe("dollar format test", () => {
+  it("converts a number to a dollar-formatted string", () => {
+    expect(dollarFormat(1000000)).toBe("$1,000,000");
+  });
 });
```

Comment:
i'm thinking we organize vitetest tests like these to be close to the modules they relate to, so maybe move these tests to a new test file like `src/lib/utils/index.test.js`. This way we can keep all the relevant stories and tests co-located with the components and modules they relate to.